tests/qat/test_qasm.py

```diff
@@ -603,6 +604,37 @@ def test_sech(self):
         assert np.isclose(pulses[1].amp, 0.5)
         assert np.isclose(pulses[1].std_dev, 20e-9)
 
+    @pytest.mark.parametrize(
+        "gate_tup", get_default_qasm3_gate_qasms(), ids=lambda val: val[-1]
+    )
+    def test_default_gates(self, gate_tup):
+        """Check that each default gate can be parsed individually."""
+        N, gate_string = gate_tup
+        if gate_string.startswith("cu("):
+            pytest.skip("CU gate is not yet supported.")
```

Comment:
I'll take a look at this later, still trying to understand what Qiskit is doing 🧐